website/docs/d/nsxt_firewall.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_firewall"
+sidebar_current: "docs-vcd-resource-nsxt-firewall"
+description: |-
+  Provides a data source to read NSX-T Firewall configuration of an edge gateway. Firewalls allow 
+  user to control the incoming and outgoing network traffic to and from an NSX-T Data Center 
+  edge gateway, you create firewall rules.
+---
+
+# vcd\_nsxt\_firewall
+
+Supported in provider *v3.3+* and VCD 10.1+ with NSX-T backed Edge Gateways.
+
+Provides a data source to read NSX-T Firewall configuration of an edge gateway. Firewalls allow 
+user to control the incoming and outgoing network traffic to and from an NSX-T Data Center 
+edge gateway, you create firewall rules.
```

Comment:
"you create firewall rules" ending seems abrupt. Please see if the paragraph can be clarified, rephrased or extended.

Nit: if we're capitalizing "Data Center", we should do same with "Edge Gateway".

Reply:
Yes. I actually copied some of these parts from docs and probably got confused. True about capitalization - I did that now.